packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5489,17 +5489,19 @@ export default class Meeting extends StatelessWebexPlugin {
           },
         };
 
+        this.unpublishStreams([
+          this.mediaProperties.audioStream,
+          this.mediaProperties.videoStream,
+          this.mediaProperties.shareAudioStream,
+          this.mediaProperties.shareVideoStream,
+        ]);
+
         this.cleanupLocalStreams();
 
         this.mediaProperties.setMediaDirection(mediaSettings.mediaDirection);
         this.mediaProperties.unsetRemoteMedia();
 
-        // when a move to is intiated by the client , Locus delets the existing media node from the server as soon the DX answers the meeting
-        // once the DX answers we establish connection back the media server with only receiveShare enabled
-        // @ts-ignore - reconnectMedia does not accept any argument
-        await this.reconnectionManager.reconnectMedia(mediaSettings).then(() => {
```

Comment:
The call to reconnectMedia() was creating a new confluence in the backend (because the reconnection triggers a call to Locus /media API with a new SDP offer and an empty `mediaId`). It would be good to confirm with the backend teams (Locus, Linus) that we don't need to do that for this moveTo flow

Reply:
update for anyone reading this: after various attempts and discussions, the conclusion is that we need to stop the old media connection and cleanup everything related to that including the local streams and then call addMedia() again and make sure that TURN discovery is done if it was done during the initial meeting join.